support-frontend/assets/components/subscriptionsProductDescription/subscriptionsProductDescription.tsx

```diff
@@ -75,14 +58,24 @@ function SubscriptionsProductDescription({
 }: PropTypes): JSX.Element {
 	return (
 		<div>
-			<h2 className="subscriptions__product-title">{title}</h2>
-			{offer && <h3 className="subscriptions__sales">{offer}</h3>}
+			<h2
+				css={[subscriptionTitle, isFeature ? subscriptionTitleFeature : css``]}
+			>
+				{title}
+			</h2>
 			{offer && (
-				<h3 className="subscriptions__product-subtitle--small">{subtitle}</h3>
-			)}
-			{!offer && (
-				<h3 className="subscriptions__product-subtitle--large">{subtitle}</h3>
+				<h3
+					css={[
+						subscriptionOffer,
+						isFeature ? subscriptionOfferFeature : css``,
```

Comment:
I think emotion ignores false values, so a cleaner way would be:
```suggestion
						isFeature && subscriptionOfferFeature,
```

Reply:
Ah ok, updated throughout this PR's code. Thanks